doc/release-process.md

```diff
@@ -307,9 +307,9 @@ bitcoin.org (see below for bitcoin.org update instructions).
   - First, check to see if the Bitcoin.org maintainers have prepared a
     release: https://github.com/bitcoin-dot-org/bitcoin.org/labels/Core
 
-      - If they have, it will have previously failed their Travis CI
+      - If they have, it will have previously failed their Cirrus CI
```

Comment:
bitcoin(core).org is still using Travis
```suggestion
      - If they have, it will have previously failed their CI
```

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/20691#issuecomment-747767182).